index.html

```diff
@@ -11,105 +11,105 @@
 
     <div class="main-grid-item">
       <div class="seo-icon"></div>
       <h3 class="red-color">SEO</h3>
-      <p class="white-color">Learn from tutors from various countires on how to implement Search Engine Opmitization</p>
+      <p class="seo-padding white-color">Learn from tutors from various countires on how to implement Search Engine Opmitization</p>
     </div>
 
     <div class="main-grid-item">
       <div class="machine-learning-icon"></div>
-      <h3 class="red-color">Machine Learning</h3>
-      <p class="white-color">Learn from tutors from various countires on how to develop Artificial Intelligence systems</p>
+      <h3 class="red-color">Machine<br>Learning</h3>
+      <p class="machine-learning-padding white-color">Learn from tutors from various countires on how to develop AI</p>
     </div>
     
      <div class="main-grid-item">
-      <div class="blockchain-icon"></div>
-      <h3 class="red-color">Blockchain Technology</h3>
-      <p class="white-color">Learn from tutors from various countires on how to develop Blochchiain Platforms</p>
+      <div class=" blockchain-icon"></div>
+      <h3 class="red-color">Blockchain<br>Technology</h3>
+      <p class="blockchain-padding white-color">Learn from tutors from various countires on how to develop Blochchiain Platforms</p>
      </div>
-     
-      <div class="main-grid-item">
-        <div class="crpyto-icon"></div>
-        <h3 class="red-color">Crypto Analysis</h3>
-        <p class="white-color">Learn from tutors from various countires on how to develop effectively analyse the crypto market improved trading strategy</p>
-      </div>
 
-      
       <div class="main-grid-item">
         <div class="editing-icon"></div>
         <h3 class="red-color">Editing</h3>
-        <p class="white-color">Learn from tutors from various countires on how to edit your publications</p>
+        <p class="editing-padding white-color">Learn from tutors from various countires on how to edit your publications</p>
       </div>
-      <button class="register-button red-background" type="button">Register for Next Cohorts</button>       
-   </section> 
-  
-    <section class="featured-speakers">
-    <h2 class="align-center">Featured Tutors</h2>
-    <div class="flex-container display-flex">
-      <div class="chinedu-image"></div>
-      <div><h4>Chinedu Ezea</h4>
-      <p>Chinedu is a doctor of philosophy at Nnamdi Azikiwe University</p>
-      <p>He is a mathematician and a professional editors with several publications in high impact factor journals</p>
-    </div>
-  </div>
-  </section>
+      </div>
+      <div class="align-center">
+      <button class="mobile-btn register-button red-background" type="button">Register for Next Cohorts</button>       
+      <a class="regiter-link white" href="">Register for Next Cohorts</a>
+    </div>    
+    </section>  
+    <section id="tutor">
+    <h2 class="align-center">Featured Tutors</h2><hr class="red-background tutor-dash">
```

Comment:
- [ ] [OPTIONAL] The original design has a squared images, please consider using the same styles so your design is almost identical to the CC design

|Your design|CC design|
|-|--|
|![Screenshot_20220526_194558](https://user-images.githubusercontent.com/25227022/170555727-192f95a3-c492-45e0-b8d8-d74b01579d75.png)|![Screenshot_20220526_194612](https://user-images.githubusercontent.com/25227022/170555759-964325d6-cd98-4939-a072-c1bf6e1afe7d.png)|

Reply:
Done